src/v0/sources/auth0/mapping.json

```diff
@@ -65,6 +65,6 @@
   },
   {
     "sourceKeys": "type",
-    "destKeys": "source_type"
+    "destKeys": ["source_type", "context.source_type"]
```

Comment:
```suggestion
    "destKeys": ["source_type", "properties.source_type"]
```

Reply:
Better to add this under properties, since we are already mapping other fields from Auth0 to RudderStack in same object.